src/features/governance/hooks/useProposalQuorum.tsx

```diff
@@ -15,19 +19,44 @@ interface ParticipationParameters {
 
 export function useProposalQuorum(propData?: MergedProposalData) {
   const participationParameters = useParticipationParameters();
+  const thresholds = useThresholds(propData?.proposal);
 
-  if (!propData?.proposal?.networkWeight) return undefined;
+  if (!propData || !propData.proposal) return;
+  if (!thresholds || !participationParameters) return;
 
-  return BigInt(
-    BigNumber(propData.proposal.networkWeight.toString())
-      .times(participationParameters.baseline)
-      .times(participationParameters.baselineQuorumFactor)
-      .toFixed(0),
+  // https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/governance/Governance.sol#L1567
+  let quorum = new BigNumber(participationParameters.baseline).times(
```

Comment:
this would change depending on what parameters constitutional parameters are being called.  (unless that is something else)

Reply:
ok i see that is what thresholds are!